sap_business_document_processing/document_information_extraction_client/constants.py

```diff
@@ -36,7 +36,9 @@
 
 API_HEADER_ACCEPT = 'accept'
 
+CONTENT_TYPE_PDF = 'application/pdf'
 CONTENT_TYPE_PNG = 'image/png'
+CONTENT_TYPE_UNKNOWN = 'unknown'
```

Comment:
I think we should extend this list of constants to all of the supported content types. I.e. add jpeg, tiff and excel content types? https://help.sap.com/viewer/5fa7265b9ff64d73bac7cec61ee55ae6/SHIP/en-US/9e6a56d37ad1463fb3b85eed5d962a3c.html

Reply:
Originally, we didn't want to add any constants that aren't used within the library, but I think we could add them